packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -83,7 +82,7 @@ window.initX = {
 window.initX = function () {
   return {
     instance: 'my-store',
-    env: location.href.includes('.pre.') ? 'staging' : 'live',
+    ...(location.href.includes('.pre.') ? { env: 'staging' } : {}),
```

Comment:
Just sending `undefined` works and it is a simpler technique for the users to understand (I guess). We still have to deal with the env selector in our demo though. It doesn't accept `undefined` values, and there are some endpoints that trigger a 400 when used in production.

```suggestion
    env: location.href.includes('.pre.') ? 'staging' : undefined,
```